packages/terra-table/src/subcomponents/Cell.jsx

```diff
@@ -384,12 +390,13 @@ function Cell(props) {
   // Determine table cell header attribute values
   const cellLeftEdge = (columnIndex < columnContext.pinnedColumnOffsets.length) ? columnContext.pinnedColumnOffsets[columnIndex] : null;
   const CellTag = isRowHeader ? 'th' : 'td';
-  const columnHeaderId = `${tableId}-${columnId}-headerCell`;
+  const columnHeaderId = `${tableId}-${columnId.split('_')[0]}-headerCell`;
   const rowHeaderId = !isRowHeader && rowHeaderIndex !== -1 ? `${tableId}-rowheader-${rowId} ` : '';
   const sectionHeaderId = sectionId ? `${tableId}-${sectionId} ` : '';
 
   let columnHighlight = {};
-  if (columnHighlightColor) {
+  // Column highlighting is not supported for multiple column spans
```

Comment:
If column highlighting is not supported for column with a colspan, why not just update the attributes of the column shape such that there is no column highlighting in this scenario.  We should not have to update the cell component.  The cell component doesn't need to know how many columns are spanned.

Reply:
Updated the logic.